indy_client/test/cli/test_send_get_schema.py

```diff
@@ -1,112 +1,109 @@
-
-    do('send GET_SCHEMA dest={} name=Degree version=1.0'.format(
-        trusteeCli.activeDID), expect=SCHEMA_FOUND, within=5)
-
-
-def test_send_get_schema_as_alice(
-        be, do, poolNodesStarted, trusteeCli, send_schema, aliceCli):
-
-    be(aliceCli)
-    do('send GET_SCHEMA dest={} name=Degree version=1.0'.format(
-        trusteeCli.activeDID), expect=SCHEMA_FOUND, within=5)
-
-
-def test_send_get_schema_fails_with_invalid_name(
-        be, do, poolNodesStarted, trusteeCli, send_schema):
-
-    do('send GET_SCHEMA dest={} name=invalid version=1.0'.format(
-        trusteeCli.activeDID), expect=SCHEMA_NOT_FOUND, within=5)
-
-
-def test_send_get_schema_fails_with_invalid_dest(
-        be, do, poolNodesStarted, trusteeCli, send_schema):
-
-    uuid_identifier = createUuidIdentifier()
-    do('send GET_SCHEMA dest={} name=invalid version=1.0'.format(
-        uuid_identifier), expect=SCHEMA_NOT_FOUND, within=5)
-
-
-def test_send_get_schema_fails_with_invalid_version(
-        be, do, poolNodesStarted, trusteeCli, send_schema):
-    do('send GET_SCHEMA dest={} name=Degree version=2.0'.format(
-        trusteeCli.activeDID), expect=SCHEMA_NOT_FOUND, within=5)
-
-
-def test_send_get_schema_fails_with_invalid_version_syntax(
-        be, do, poolNodesStarted, trusteeCli, send_schema):
-
-    with pytest.raises(AssertionError) as excinfo:
-        do('send GET_SCHEMA dest={} name=Degree version=asdf'.format(
-            trusteeCli.activeDID), expect=SCHEMA_NOT_FOUND, within=5)
-    assert(INVALID_SYNTAX in str(excinfo.value))
-
-
-def test_send_get_schema_fails_without_version(
-        be, do, poolNodesStarted, trusteeCli, send_schema):
-
-    with pytest.raises(AssertionError) as excinfo:
-        do('send GET_SCHEMA dest={} name=Degree'.format(trusteeCli.activeDID),
-            expect=SCHEMA_NOT_FOUND, within=5)
-    assert(INVALID_SYNTAX in str(excinfo.value))
-
-
-def test_send_get_schema_fails_without_name(
-        be, do, poolNodesStarted, trusteeCli, send_schema):
-
-    with pytest.raises(AssertionError) as excinfo:
-        do('send GET_SCHEMA dest={} version=1.0'.format(trusteeCli.activeDID),
-            expect=SCHEMA_NOT_FOUND, within=5)
-    assert(INVALID_SYNTAX in str(excinfo.value))
-
-
-def test_send_get_schema_fails_without_dest(
-        be, do, poolNodesStarted, trusteeCli, send_schema):
-
-    with pytest.raises(AssertionError) as excinfo:
-        do('send GET_SCHEMA name=Degree version=1.0',
-            expect=SCHEMA_NOT_FOUND, within=5)
-    assert(INVALID_SYNTAX in str(excinfo.value))
+from indy_node.test.schema.helper import write_schema, prepare_get_schema_and_send
+from indy.ledger import parse_get_schema_response
+
+
+def test_send_get_schema_succeeds(looper, sdk_pool_handle,
+                                  sdk_wallet_trust_anchor):
+    _, issuer_id = sdk_wallet_trust_anchor
+
+    schema_name = "Faber"
+    schema_version = "1.8"
+    schema_attr = ["attribute1"]
+
+    write_schema(
```

Comment:
I think, it would be nice to make a fixture sended_schema (like send_schema in removed tests), instead of writing schema every time